conf/supervisord.conf

```diff
@@ -13,21 +13,30 @@ stderr_events_enabled=true
 autostart = %(ENV_ENABLE_NGINX)s
 
 [program:web]
-command=newrelic-admin run-program pserve conf/app.ini
+command=newrelic-admin run-program gunicorn --paste conf/production.ini --config conf/gunicorn.conf.py
 stdout_logfile=NONE
 stderr_logfile=NONE
 stdout_events_enabled=true
 stderr_events_enabled=true
 autostart = %(ENV_ENABLE_WEB)s
 
 [program:websocket]
-command=pserve %(ENV_WEBSOCKET_CONFIG)s
+command=newrelic-admin run-program gunicorn --paste conf/websocket.ini --config conf/gunicorn-websocket.conf.py
 stdout_logfile=NONE
 stderr_logfile=NONE
 stdout_events_enabled=true
 stderr_events_enabled=true
 autostart = %(ENV_ENABLE_WEBSOCKET)s
 
+[program:websocket-monolithic]
+command=newrelic-admin run-program gunicorn --paste conf/websocket-monolithic.ini --config conf/gunicorn-websocket-monolithic.conf.py
+stdout_logfile=NONE
+stderr_logfile=NONE
+stdout_events_enabled=true
+stderr_events_enabled=true
+autostart = %(ENV_ENABLE_WEBSOCKET_MONOLITHIC)s
+process_name = websocket
```

Comment:
Previously there was a single `websocket` "program" in `supervisord.conf` and a `%(ENV_WEBSOCKET_CONFIG)s` envvar was used to make the websocket config file vary between different QA/staging/production environments: `conf/websocket-separate.ini` (now renamed to just `websocket.ini`) or `conf/websocket-monolithic.ini`.

This will no longer work now that we have separate PasteDeploy and Gunicorn config files. You'd need two separate envvars `WEBSOCKET_PASTEDEPLOY_CONFIG` and `WEBSOCKET_GUNICORN_CONFIG` or some other scheme.

I've changed this to instead define two separate `websocket` and `websocket-monolithic` processes in `supervisord.conf`. We'll then set the `ENABLE_WEBSOCKET` and `ENABLE_WEBSOCKET_MONOLITHIC` envvars as necessary in each environment.

These `ENABLE_{PROGRAM}` envvars are a pattern that's available for all our programs/processes in all our repos (in fact they're already used to turn off the websocket process in h's QA and production environments).

The `WEBSOCKET_CONFIG` envvar will no longer be used.